rust/rsc/src/rsc/blob_store_service.rs

```diff
@@ -39,6 +39,9 @@ pub async fn fetch_unreferenced_blobs(
     //                 .union( *OTHER STUFF* ));
     //                 .take().
 
+    // Limit = 16k as the query is also subject to parameter max.
+    // Blob has 4 params so (2^16)/4 = 16384. Also generally best to chunk blob eviction
```

Comment:
Is there any way to dynamically calculate this, or at least to explicitly list the params rather than using `SELECT *`?  I know we're quite unlikely to change the schema, but if we ever do for some reason we risk introducing a bug which might only be caught after rollout if the tests aren't big enough.

Reply:
There isn't really a good way to do that no, We'd need some kind of language level inspection to determine how many parameters the struct has and I don't know if that is possible without some `macro!` magic that I don't really want to dive into.

TBH the better solution will be just just update SeaORM once they update their sqlx dependency. Newer version of sqlx allow for 2^32 parameter which we are waaay less likely to hit